examples/events.py

```diff
@@ -67,11 +76,10 @@ def future_loading(t, x=None):
             i = 3
         return {'i': i}
     
-    first_output = {'t': 18.95, 'v': 4.183}
     m = MyBatt()
 
     # 2b: Simulate to threshold
-    (times, inputs, states, outputs, event_states) = m.simulate_to_threshold(future_loading, first_output, threshold_keys=['EOD'], print = True)
+    (_, _, _, _, event_states) = m.simulate_to_threshold(future_loading, threshold_keys=['EOD'], print = True)
```

Comment:
For clarity, it might be nice to leave (times, inputs, states, outputs, event_states) as a reminder of what the output is, unless the code needs it to be shorter

Reply:
```suggestion
    (times, inputs, states, outputs, event_states) = m.simulate_to_threshold(future_loading, threshold_keys=['EOD'], print = True)
```